netket/sampler/metropolis.py

```diff
@@ -77,7 +77,8 @@ def reset(
             sampler_state: The current state of the sampler. Should not modify it.
 
         Returns:
-            An Optional State. Must return the same type of `sampler_state.rule_state`.
+            An Optional State. Must return the same type of :py:meth:`sampler_state.rule_state`.
```

Comment:
This only says "optional state", but not in which case a state is returned and when it isn't. Maybe this could be clarified?

Reply:
It 
```suggestion
           A new, resetted, state of the rule. This returns the same type of :py:meth:`sampler_state.rule_state` and might be None.
```

Reply:
addressed